proposals/0000-ghc-extensions.rst

```diff
@@ -0,0 +1,120 @@
+
+  The secretary also creates a PR with a proposal saying (roughly) 
+   
+    GHC202x contains the following extensions in addition to those in GHC202(x-1):
+    
+    * (none yet)
+    
+    and removes these extensions
+   
+    * (none yet)
+
+* In order to gather data on the criterium “widespread usage”, the secretary creates a tally of which extensions are used how often on hackage.
+
+* In order to gather data on the criterium “community spport”, the secretary runs a public poll on a suitable platform for one week where anyone
+  can vote in favor or against the inclusion of a given extension.  
+
+* Within two weeks of the start of the process, every committee member is expected to send an initial list of which extensions they expect to be in GHC202x to the mailing list.
+   
+  Committe members are expected to take the Hackage statistics and the community vote into account.
+   
+  These mails may contain justifications for why a certain extension is or is not included, but not required (or even expected).
+
+  After these two weeks, the PR is continuously updated by the secretary to reflect the *current* tally of votes: An extension is included if it is listed by at least ⅔ (rounded up) of committee members.
+
+* Within four weeks of the start of the process, committee members can change their vote (by email to the list).
+
+  It is absolutely ok to change one’s mind based on the explanations in the other members’ emails, or the general comments on the PR.
+   
+  Long discussions of individual extensions are discouraged at this point. If there is controversy around an extension, it is a strong sign that it should simply not be included.
+
+* After these four weeks, the proposal with the current tally gets accepted by the secretary, and defines GHC202x
+
+Cadence
+^^^^^^^
+
+Likely, the first iteration of this process will be vastly different from the following ones: The first one is expected to add a large number of uncontroversial extensions; so the next iteration will likely only make a smaller, but more controversial change.
+
+Therefore, this proposal does *not* commit to a fixed cadence. Instead, 6 months after the first release of a version of GHC that supports a GHC20XX set, we evaluate the outcome, the process, and the perceived need of a next release. At that time we will refine the processes, if needed, and set a cadence.
+
+Costs and Drawbacks
+-------------------
+
+The implementation cost seems small.
+
+The cost of a GHC20xx extension is that, upon reading a file with ``{-# LANGUAGE GHC20xx #-}``, the reader does not immediatelly know the set of enabled extensions; this may hamper readability of code.
+
+The costs of this process is that it binds volunteer time, and there is is risk of unpleasant, heated debates, because everybody has opinions. The process tries to err on the conversative side and rather add too few than too many extensions.
```

Comment:
```suggestion
The costs of this process is that it binds volunteer time, and there is is risk of unpleasant, heated debates, because everybody has opinions. The process tries to err on the conservative side and rather add too few than too many extensions.
```